test/third_party/install.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
+
+set -v
+
+
+BASE_DIR=$(readlink -f $(dirname "$0"))
+
+export PATH="$BASE_DIR/google-cloud-sdk/bin:$PATH"
+
+GCLOUD_SDK_URL="https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-145.0.0-linux-x86_64.tar.gz"
+GCLOUD_SDK_TGZ="$BASE_DIR/google-cloud-sdk.tar.gz"
+GCLOUD_PATH="$(which gcloud)"
+
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+RED='\033[0;31m'
+NC='\033[0m'
+
+function win() {
+    printf "\n${GREEN}$1${NC}\n"
```

Comment:
You could use $@ instead of $1. That will cause all arguments to be output instead of just the first.
You aren't using formatting, so echo would be more appropriate.
Having a leading newline in output feels odd.